README.md

```diff
@@ -841,11 +841,13 @@ cmd
 [TOC](#TOC)
 <div id="5-6">
 
-### Unix
+### 5.6 Unix
 
-Exiv2 can be built on many Unix and Linux distros.  We actively support the Unix Distributions NetBSD and FreeBSD.
+Exiv2 can be built on many Unix and Linux distros.  With v0.27.2, we are starting to actively support the Unix Distributions NetBSD and FreeBSD.  We hope to add CI support for these platforms in v0.27.3.
 
-I am willing to support Exiv2 on commercial Unix distributions such as Solaris, AIX, HP-UX and OSF/1 provided you provide with an ssh account on your platform.  I will require super-user privileges to install software.
+I have provided notes here based on my experience with these platforms.   Feedback is welcome.
```

Comment:
[minor] We should review this kind of comments for the next release. In my opinion, the main README should contain general information about the project, and not this kind personal statements (Personal pronoun in first singular person)

Reply:
Well, it's how I express myself.  For v0.28, you (or somebody else) can rewrite the documentation if they wish.